components/hab/src/main.rs

```diff
@@ -1372,7 +1372,10 @@ where
         match status.process {
             Some(process) => (
                 process.state.to_string(),
-                process.pid.unwrap_or_default().to_string(),
+                process
+                    .pid
+                    .map(|p| p.to_string())
+                    .unwrap_or("<none>".to_string()),
```

Comment:
To me, instead of `.map(…).unwrap_or(…)`, it's slightly more evocative to use:
```rust
                    .map_or_else(|| "<none>".to_string(), |p| p.to_string()),
```

But you can take that or leave it according to what you like. Either way, I'll plan to merge this tomorrow.

Reply:
Ooooh, that's handy! Changed.